plenum/test/test_action_queue.py

```diff
@@ -17,19 +17,45 @@ def __init__(self, name):
         def start(self, loop):
             pass
 
-        async def prod(self, limit: int = None) -> int:
+        async def prod(self, limit: int=None) -> int:
             return self._serviceActions()
 
-        def meth1(self, x):
-            if 'meth1' not in self.results:
-                self.results['meth1'] = []
-            self.results['meth1'].append((x, time.perf_counter()))
+        def meth(self, meth_name, x):
+            if meth_name not in self.results:
+                self.results[meth_name] = []
+            self.results[meth_name].append((x, time.perf_counter()))
 
     with Looper() as looper:
         q1 = Q1('q1')
+
+        q1.meth1 = partial(q1.meth, 'meth1')
```

Comment:
Can we make the test to look more like a test, and separate it into test cases (like scheduling actions, cancelling actions, re-scheduling the same actions, etc.)

Reply:
done